tests/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class UnmockedError(Exception):
+    def __init__(self):
+        super(Exception, self).__init__(
```

Comment:
This signature for `__init__` is incorrect; it should include a message parameter. Also, `super` must be called with the child class, not the parent class.
```python
def __init__(self, message='No mocking exists...'):
    super(UnmockedError, self).__init__(message)
```